toponetx/utils/normalization.py

```diff
@@ -116,22 +114,23 @@ def compute_kipf_adjacency_normalized_matrix(
     return A_opt_normalized
 
 
-def compute_xu_asymmetric_normalized_matrix(B, is_sparse=True):
-    """
-    Compute Xu's normalized asymmetric matrix.
+def compute_xu_asymmetric_normalized_matrix(
+    B: Union[ndarray, csr_matrix], is_sparse: bool = True
```

Comment:
Use union type expression please:
```suggestion
    B: np.ndarray | csr_matrix, is_sparse: bool = True
```